### PR TITLE
update japanese doc

### PR DESCRIPTION
**Pull request by butaosuinu**
- translate updated docs
- change the link to Buddhist Era wikipedia to japanese wikipedia

#171

### CODE REVIEW

**Comment by iamkun** [comment]
THX

File: docs/ja/API-reference.md [hunk 1]
```diff
@@ -482,49 +480,52 @@ dayjs().isAfter(dayjs()); // false
 dayjs().isAfter(dayjs(), 'year'); // false
 ```
 
-### Is a Dayjs `.isDayjs(compared: any)`
+#### Is a Dayjs
```
Comment (iamkun) [inline docs/ja/API-reference.md]: We should update the table of contents accordingly, to ensure the anchor works.

Comment (butaosuinu) [reply docs/ja/API-reference.md]: Oops, fixed.

**Pull request merged**